### DISCUSSION TITLE
The Sidebar Menu Bugs on IE

### DISCUSSION THREAD

**Discussion by CharlieYAN** [ColorlibHQ/AdminLTE#3531] [Show and tell]
I am using the latest RC edition, and I found bugs when expanding the sidebar menu 'nav-treeview' item. 

When setting the css class  ".nav-collapse-hide-child" on the  <ul> element, then expanding the menu item, the menu items text overlaps. 

![image](https://user-images.githubusercontent.com/57309053/112357895-d48afd00-8d0a-11eb-8221-fc5b13fd0dfb.png)

**Comment by danny007in** [comment]
use latest `jquery`

**Reply by danny007in** [reply]
use latest v3.1 Adminlte

**Reply by CharlieYAN** [reply]
The bug is not fixed on the latest v3.1 Adminlte. 
The same problem existing when using the latest IE 11 that Adminlte announced to support it in README file.

Hope anyone to fix it if possible.

![image](https://user-images.githubusercontent.com/57309053/112457398-b074fd80-8d96-11eb-907c-1894b77dd668.png)

**Reply by danny007in** [reply]
use Edge Browser till the issue to resolve

**Reply by danny007in** [reply]
and create this as issue
